src/openzeppelin/tests/test_reentrancyguard.cairo

```diff
@@ -1,43 +1,107 @@
-use openzeppelin::tests::mocks::reentrancy_attacker_mock::ReentrancyAttackerMock;
+use openzeppelin::security::reentrancyguard::ReentrancyGuard;
 use openzeppelin::tests::mocks::reentrancy_mock::ReentrancyMock;
+use openzeppelin::tests::mocks::reentrancy_mock::IReentrancyMockDispatcher;
+use openzeppelin::tests::mocks::reentrancy_mock::IReentrancyMockDispatcherTrait;
+use openzeppelin::tests::mocks::reentrancy_attacker_mock::Attacker;
 
+use array::ArrayTrait;
+use core::result::ResultTrait;
+use option::OptionTrait;
+use starknet::class_hash::Felt252TryIntoClassHash;
 use starknet::ContractAddress;
+use traits::TryInto;
+
+fn deploy_mock() -> IReentrancyMockDispatcher {
+    let calldata = ArrayTrait::<felt252>::new();
+    let (address, _) = starknet::deploy_syscall(
+        ReentrancyMock::TEST_CLASS_HASH.try_into().unwrap(), 0, calldata.span(), false
+    )
```

Comment:
thinking we could abstract deployments to make them easier to read?

```suggestion
    let address = deploy(ReentrancyMock, calldata);
```

```cairo
fn deploy(contract, calldata) -> ContractAddress {
    let (address, _) = starknet::deploy_syscall(
        contract::TEST_CLASS_HASH.try_into().unwrap(), 0, calldata.span(), false
    )
    address
}
```

or something similar

Reply:
Great idea!